plugin/generator.go

```diff
@@ -19,26 +19,36 @@ import (
 	"github.com/docker/docker/client"
 )
 
-var proxyServiceTasks bool
-var caddyNetworks map[string]bool
+var labelPrefixFlag string
+var proxyServiceTasksFlag bool
+var isTrue = regexp.MustCompile("^(true|True|TRUE|yes|Yes|YES|1)$")
 
 func init() {
-	flag.BoolVar(&proxyServiceTasks, "proxy-service-tasks", false, "Proxy to service tasks instead of VIP")
+	flag.StringVar(&labelPrefixFlag, "docker-label-prefix", "caddy", "Prefix for Docker labels")
+	flag.BoolVar(&proxyServiceTasksFlag, "docker-proxy-service-tasks", false, "Proxy to service tasks instead of VIP")
 }
 
-
-func getCaddyLabelPrefix() string {
+func getLabelPrefix() string {
 	if val := os.Getenv("CADDY_DOCKER_LABEL_PREFIX"); val != "" {
 		return val
+	} else {
+		return labelPrefixFlag
 	}
+}
 
-	return "caddy"
+func getProxyServiceTasks() bool {
+	if val := os.Getenv("CADDY_DOCKER_PROXY_SERVICE_TASKS"); isTrue.MatchString(val) {
```

Comment:
Same thing here, I think we could compute the value of ProxyServiceTasks only once, during init.